info.json

```diff
@@ -15,27 +15,81 @@
     ],
     "prerequisite": null,
     "publisher": "Fortinet",
-    "description": "Solution pack demonstrates GDPR process",
-    "help": "https:\/\/github.com\/fortinet-fortisoar\/solution-pack-gdpr-framework\/blob\/develop\/README.md",
+    "description": "This solution pack demonstrates procedures involved in GDPR",
+    "help": "https://github.com/fortinet-fortisoar/solution-pack-gdpr-framework/blob/develop/README.md",
     "category": [],
-    "supportInfo": "support@fortinet.com",
+    "supportInfo": "",
     "iconLarge": null,
     "fsrMinCompatibility": "7.2.0",
-    "date": "2022-03-15T06:07:50+00:00",
-    "exported_from": "tenant.fortisoar.com",
-    "exported_by": "CS Admin",
+    "date": "2022-04-19T12:47:14+00:00",
     "contents": {
-        "playbooks": [
+        "picklistNames": [
+            {
+                "name": "Contact Title"
+            },
+            {
+                "name": "DataBreachType"
+            },
+            {
+                "name": "GDPREUCountryList"
+            },
+            {
+                "name": "IncidentType"
+            },
+            {
+                "name": "TaskType"
+            },
             {
-                "name": "GDPR Framework"
+                "name": "YesNo Picklist"
             }
         ],
-        "globalVariables": [
+        "modules": {
+            "contact": {
+                "name": "Contacts",
+                "apiName": "contact"
+            },
+            "": {
+                "name": "Incidents",
```

Comment:
Incident will be part of SFSP. Do we still need to add it here?

Reply:
Yes, in this solution pack i have added some new fields in incident modules